### PR TITLE
SWPY-1076 remove avaliable_in_production/is_production_environment

### PR DESCRIPTION
**Pull request by Kareeeeem**
Removed all references to these, including the docs and tests

### CODE REVIEW

File: spynl/main/mail.py
Comment (koraa) [inline spynl/main/mail.py:10]: Line 48 seems to still contain a reference to the production environment

Comment (koraa) [reply spynl/main/mail.py:10]: "that if Spynl is not in a production environment, mail is sent to a dummy"

Comment (koraa) [reply spynl/main/mail.py:10]: Please also check line 158 again – I am not sure whether this still makes sense

**Review by koraa — APPROVED** [review]
Looking good! ++1

File: spynl/main/mail.py
Comment (Kareeeeem) [reply spynl/main/mail.py:10]: I think someone who knows what the deal is with dummy mailer should have a look here. Not sure what the intended behavior is.

Comment (RosanneZe) [reply spynl/main/mail.py:10]: The dummy mailer is used in the unit tests. But the line at 158 can probably go (I'll put a comment at the line, because I got a bit confused by these comments at the start of the file)

File: spynl/main/mail.py
Comment (RosanneZe) [inline spynl/main/mail.py]: This complete if statement and content of if statement needs to go. (This mail would in production etc...) The way it is now, this will be send to every email in the production environment.

**Review by RosanneZe — CHANGES_REQUESTED** [review]
There is an if statement that needs to be removed, looks good otherwise. 
Also, please test the emailing before you merge. (serve locally and send a pwd reset key or something like that, or if you can't get the local emailing to work, build in latest and test there)

**Review by o-m-n-i — APPROVED** [review]
looks good to me

**Review by RosanneZe — APPROVED** [review]

**Pull request merged**
